lib/kafkalib/topic.go

```diff
@@ -65,20 +66,24 @@ func (t *TopicConfig) String() string {
 		t.Database, t.Schema, t.TableName, t.Topic, t.IdempotentKey, t.CDCFormat, t.DropDeletedColumns)
 }
 
-func (t *TopicConfig) Valid() bool {
+func (t *TopicConfig) Validate() error {
```

Comment:
Returning an error instead of a boolean so that we can be more descriptive when it fails to validate.